integration/cloudbuild/prepare.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -x
+set -e
+
+git clone --depth=1 https://github.com/googleapis/googleapis.git "$GOPATH/src/github.com/googleapis/googleapis"
+go install \
+    github.com/golang/protobuf/proto \
```

Comment:
Again, this installs some "latest" version (master?), which might be why there is an etcd/protobuf panic.